openeo-geotrellis/src/main/scala/org/openeo/geotrellis/package.scala

```diff
@@ -89,4 +90,32 @@ package object geotrellis {
 
     Region.of(regionName)
   }
+
+  def toSigned(cellType: CellType): CellType = {
+    cellType match {
+      case UByteCellType => ByteCellType
+      case UByteConstantNoDataCellType => ByteConstantNoDataCellType
+      case c: UByteUserDefinedNoDataCellType => ByteUserDefinedNoDataCellType(c.noDataValue)
+      case UShortCellType => ShortCellType
+      case UShortConstantNoDataCellType => ShortConstantNoDataCellType
+      case c: UShortUserDefinedNoDataCellType => ShortUserDefinedNoDataCellType(c.noDataValue)
+      case FloatConstantNoDataCellType => cellType
+      case ShortConstantNoDataCellType => cellType
+      case BitCellType => cellType
+      case ByteConstantNoDataCellType => cellType
+      case ByteCellType => cellType
+      case _: ByteUserDefinedNoDataCellType => cellType
+      case ShortCellType => cellType
+      case _: ShortUserDefinedNoDataCellType => cellType
+      case IntConstantNoDataCellType => cellType
+      case IntCellType => cellType
+      case _: IntUserDefinedNoDataCellType => cellType
+      case FloatCellType => cellType
+      case _: FloatUserDefinedNoDataCellType => cellType
+      case DoubleConstantNoDataCellType => cellType
+      case DoubleCellType => cellType
+      case _: DoubleUserDefinedNoDataCellType => cellType
+      case _ => throw new IllegalArgumentException("Can convert to unsigned equivalent: '" + cellType.getClass.getName + "'.")
```

Comment:
Typo: Cannot?

String interpolation makes it easier to read:

```scala
s"Cannot ... '${cellType.getClass.getName}'."
```

Reply:
Indeed, fixed!